### DISCUSSION TITLE
Checkpoint file

### DISCUSSION THREAD

**Discussion by qifeng22** [open-mmlab/mmengine#1293] [General]
When I trained the sassd model, I did not generate a checkpoint file

**Comment by zhouzaida** [comment]
Hi, have you set the interval (the default value is -1, which means "never") in CheckpointHook?

https://mmengine.readthedocs.io/en/latest/common_usage/set_interval.html#setting-the-interval-for-saving-checkpoints

**Comment by qifeng22** [comment]
Thank for your reply. It  worked when set the interval value is 1.
